src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/runWithEnvVariables.groovy

```diff
@@ -25,6 +26,8 @@ podTemplate(
         echo OUTSIDE_CONTAINER_ENV_VAR_FROM_SECRET = $CONTAINER_ENV_VAR_FROM_SECRET or `echo $CONTAINER_ENV_VAR_FROM_SECRET | tr [a-z] [A-Z]`
         echo OUTSIDE_POD_ENV_VAR = $POD_ENV_VAR
         echo OUTSIDE_POD_ENV_VAR_FROM_SECRET = $POD_ENV_VAR_FROM_SECRET or `echo $POD_ENV_VAR_FROM_SECRET | tr [a-z] [A-Z]`
+        echo OUTSIDE_EMPTY_POD_ENV_VAR = $EMPTY_POD_ENV_VAR
+        echo OUTSIDE_EMPTY_POD_ENV_VAR_FROM_SECRET = $EMPTY_POD_ENV_VAR_FROM_SECRET or `echo $EMPTY_POD_ENV_VAR_FROM_SECRET | tr [a-z] [A-Z]`
```

Comment:
Do not really need to `tr` here, could just say something like

```sh
echo "OUTSIDE_EMPTY_POD_ENV_VAR_FROM_SECRET = '$EMPTY_POD_ENV_VAR_FROM_SECRET'"
```

which should print

```
OUTSIDE_EMPTY_POD_ENV_VAR_FROM_SECRET = ''
```